app/workflows/schemas/task-list.json

```diff
@@ -16,6 +16,9 @@
               "type": "object",
               "properties": {
                 "title": { "type": "string" },
+                "hint": { "type": "string" },
```

Comment:
I think I did the same with the `optional` key, do we need to declare the property as optional as well?

Reply:
The schema only has `required` and not `optional`.